cobbler/items/item.py

```diff
@@ -406,11 +406,8 @@ def kernel_options(self, options):
         :param options: The new kernel options as a space delimited list.
         :raises ValueError: In case the values set could not be parsed successfully.
         """
-        (success, value) = utils.input_string_or_dict(options, allow_multiples=True)
-        if not success:
-            raise ValueError("invalid kernel options")
-        else:
-            self._kernel_options = value
+        value = utils.input_string_or_dict(options, allow_multiples=True)
+        self._kernel_options = value
```

Comment:
Maybe these changes we should do like this?

```suggestion
        self._kernel_options = utils.input_string_or_dict(options, allow_multiples=True)
```

OTOH, aren't we loosing the nice errors messages, for each case, and switching to generic error message? Is that ok?

Reply:
At `utils.input_string_or_dict` I only see a generic error: "invalid input type" to be raised in some cases, but here we are removing different concrete errors messages for each different of attribute.

I would like we preserve that granularity if possible for the error messages, that would always help on debugging.

Reply:
Okay yes then I would need to add try-catch blocks. Let me cook something up.

Reply:
@meaksh I did cook something up. Please have a look again and resolve this if you are happy. :)

Reply:
I will assume that this is good now. Thus resolving the conversation.